internal/system/client/identity_client.go

```
@@ -342,6 +342,64 @@ func (c *IdentityClient) FetchApplicationIdentifier(applicationIdentifier, orgHa
+	}
+
+	appsFailed := func(desc string, cause error) error {
+		return errors2.NewServerError(errors2.ErrorMessage{
+			Code:        errors2.GET_APPLICATIONS_FAILED.Code,
+			Message:     errors2.GET_APPLICATIONS_FAILED.Message,
+			Description: desc,
+		}, cause)
+	}
+
+	appEndpoint := fmt.Sprintf("https://%s/t/%s/api/server/v1/applications/%s", c.BaseURL, orgHandle, appID)
+	req, err := http.NewRequest("GET", appEndpoint, nil)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)